changelog.md

```diff
@@ -1,10 +1,21 @@
 
 
+## 3.3.1 / 2020-12-17
+
+### Fixed Issues
+
+- Note: [#3722](https://github.com/tracim/tracim/issues/3722)
+- Mention: [#3927](https://github.com/tracim/tracim/issues/3927)
+- Frontend: [#3542](https://github.com/tracim/tracim/issues/3542),
+[#3809](https://github.com/tracim/tracim/issues/3809)
```

Comment:
```suggestion
- Frontend: 
    - [#3542](https://github.com/tracim/tracim/issues/3542),
    - [#3809](https://github.com/tracim/tracim/issues/3809)
```

Reply:
same as explain before (this is not the syntax for the chnagelog, at present)/

Reply:
good for me, I'll resolve the conversation.